app/views/layouts/_meta.html.erb

```diff
@@ -1,3 +1,5 @@
+<meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
I know that these two lines were copied from the `_head.html.erb`, but for the consistency of this file, I think, it will be better if we use `<%= tag.meta ... %>` instead.